docs/source/tests/pytest.rst

```diff
@@ -0,0 +1,124 @@
+============================
+About The Pytest Framework
+============================
+The pytest framework provides a feature-rich, plugin-based ecosystem that helps to easily write small as well as readable tests and it can also scale to support complex functional testing. To make sure that you can use the full functionality of pytest this section provides you some conventions and commands that are useful. If you want to get more into the whole framework you can find further information `here <https://docs.pytest.org/en/6.2.x/contents.html#toc>`__.
+As described in the :doc:`previous section </tests/testing>` pytest follows a strict naming convention for files (``test_*.py``) and methods (``def test_*()``).
```

Comment:
The `previous section` link will break if joined into one file

Reply:
all :doc: links where changed to :refs: